config/apache/cobbler_web.conf

```diff
@@ -1,6 +1,17 @@
 # This configuration file enables the cobbler web
 # interface (django version)
 
+Alias /cobbler_webui_content @@webroot@@/cobbler_webui_content
+
+# Use separate process group for wsgi
+WSGISocketPrefix /var/run/wsgi
+WSGIScriptAlias /cobbler_web @@install_data@@/share/cobbler/web/cobbler.wsgi
+WSGIDaemonProcess cobbler_web display-name=%{GROUP}
+WSGIProcessGroup cobbler_web
+WSGIPassAuthorization On
+
+<VirtualHost *:443>
```

Comment:
Why the addition of VirtualHost here?  This breaks configuration on Fedora where we inherit the ssl server from ssl.conf.  With this you would also need to add all of the SSL configuration option otherwise you end up with an HTTP server running on the HTTPS port.

Reply:
I introduced it to be able to run cobbler itself with http (which I normally do for an easier setup) and still be able to run the GUI, if needed, on the same server with https.